robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot

```diff
@@ -77,7 +77,7 @@ Create and edit level to verify fields
     ...                                  Edit the smallestAmount value to a value greater than the limit threshold
     ...                                  Run the batch job and verify the correct levels are applied.
 
-    [tags]                                  W-038641                 feature:Level              unstable            notonfeaturebranch
+    [tags]                                  W-038641    feature:Level notonfeaturebranch  api
```

Comment:
Do we need an additional space between the tags? Applies to the next one as well.